src/Equinox.MemoryStore/MemoryStore.fs

```diff
@@ -67,47 +67,41 @@ module private Token =
             version = int64 eventCount }
     let (|Unpack|) (token: StreamToken) : Token = unbox<Token> token.value
     /// Represent a stream known to be empty
-    let ofEmpty streamName initial = streamTokenOfEventCount streamName 0, initial
-    let tokenOfArray streamName (value: 'event array) = Array.length value |> streamTokenOfEventCount streamName
-    let tokenOfSeq streamName (value: 'event seq) = Seq.length value |> streamTokenOfEventCount streamName
-    /// Represent a known array of events (without a known folded State)
-    let ofEventArray streamName fold initial (events: 'event array) = tokenOfArray streamName events, fold initial (Seq.ofArray events)
-    /// Represent a known array of Events together with the associated state
-    let ofEventArrayAndKnownState streamName fold (state: 'state) (events: 'event seq) = tokenOfSeq streamName events, fold state events
```

Comment:
note this computes an incorrect token as the event count is derived from the new events, not the post-state value of the events for this stream

Reply:
also an example of how confusing egregious use of `|>` is never a good thing (I try very hard not to do that as I'm conscious of it, but I evidently thought I was being very clever)